modules/_labs/select/react/lib/SelectOption.tsx

```diff
@@ -62,12 +63,17 @@ const Option = styled('li')<SelectOptionProps>(
     } else if (focused) {
       // ...else if the option is focused, return focused styles...
       return {
-        ...activeStyles,
+        ...activeStyles(theme),
       };
     } else {
       // ...else return hover and selected (via aria-selected) styles
       const selectedStyles = {
         '&[aria-selected="true"]': {
+          // Currently, there doesn't seem to be an appropriate color for
+          // the background of a selected option in the canvas theme object.
+          // `primary.lightest` happens to match from the default theme, but
+          // I'm not sure the selected color should be based off of the primary
+          // color.
           backgroundColor: colors.blueberry100,
```

Comment:
Any thoughts on which (if any) theme color we should use for the background color of a selected option? The current color of the selected option (`blueberry100`) happens to match `primary.lightest` of the default canvas theme, but it doesn't seem like the selected color should be based off of the primary color (if anything, it should be based off of `common.focusOutline`, or maybe there needs to be a new color in `common` which can be used as a background color for selected things).

Reply:
I would use `primary.lightest` for now, but I agree with you. We'll probably do a larger audit for how theming colors are applied in the future.

Reply:
Ok, get ready for some unattractive Theming states tables. 🤣 Like purple and turquoise (I think that's what's being used in `customColorTheme`).